### ISSUE TITLE
Strapi is not free

### ISSUE THREAD

**Issue by alvarofranz** [strapi/strapi#10520]
Disappointing. Why do you repeat a thrillion times that strapi is free? It's not. 

It's a freemium model with **limits** and a paid pro version.

**Comment by pierreburgy** [comment]
Thanks for your message.

The Community Edition of Strapi is completely free, without any limitations.

On top of that, the Enterprise Edition includes additional features.

The Community Edition shouldn't be perceived as "limited". Instead, the Enterprise Edition extends and unleashes the Community Edition. Eg. creating Custom Roles is not about restricting the Community Edition but an additional feature that enables this possibility. Same thing for SSO.

That being said, we do understand it can create some confusion and will make things clearer.

Feel free to share some resources where you've seen Strapi is free but you consider it's not.

**Comment by derrickmehaffy** [comment]
Going to set this as closed as it's not a bug report or feature request but feel free to continue the discussion.

**Comment by NickUfer** [comment]
Well, then it would be cool If you then update the documentation too, as it still mentions to create a role to even being able fetch objects via the api: https://strapi.io/documentation/developer-docs/latest/guides/auth-request.html

**Comment by pierreburgy** [comment]
@NickUfer this section mentions the roles for end users, not admin users, which are two different things: https://strapi.io/documentation/developer-docs/latest/getting-started/troubleshooting.html#user-can-t-login-to-the-admin-panel

**Comment by nesjett** [comment]
Still I dont understand the following:
![image](https://user-images.githubusercontent.com/7051231/131266618-cf93407c-5ace-4774-9ad0-6d4e48942c2b.png)
![image](https://user-images.githubusercontent.com/7051231/131266625-7733ada5-1762-4f0d-903a-8351a7eec233.png)

What are those roles refering to?

**Comment by xanderakafonz** [comment]
It's explained quite well in the [documentation ](https://strapi.io/documentation/user-docs/latest/users-roles-permissions/introduction-to-users-roles-permissions.html)

**Comment by nesjett** [comment]
Yes, so still don't understand what's about saying it's open source but limiting the product to 3 roles

**Comment by derrickmehaffy** [comment]
Open source does not mean "free", see the article published on OpenSource.com: 
https://opensource.com/resources/what-open-source

A small part of that article explains the question:

> Doesn't "open source" just mean something is free of charge?

> No. This is a common misconception about what "open source" implies, and the concept's implications are not only economic.
> 
> Open source software programmers can charge money for the open source software they create or to which they contribute. But in some cases, because an open source license might require them to release their source code when they sell software to others, some programmers find that charging users money for software services and support (rather than for the software itself) is more lucrative. This way, their software remains free of charge, and they make money helping others install, use, and troubleshoot it.

And this is one of the many [common misconceptions](https://opensource.com/education/12/7/clearing-open-source-misconceptions) about open source software.

Strapi is free to use with regards to it's license: https://github.com/strapi/strapi/blob/master/LICENSE and most of the code is licensed under MIT Expat. Here is a useful OSS license => English conversion: https://tldrlegal.com/license/mit-license

With a small exception in our case, certain parts of our code are under a proprietary license which you can find here: https://github.com/strapi/strapi/blob/master/packages/strapi-admin/ee/LICENSE

Our Enterprise version as of date only has 3 paid features:

- Number of **Admin Roles** you can create
- The "Advanced RBAC" feature (which gives you more control over field level permissions and what we call [Conditionals](https://strapi.io/documentation/developer-docs/latest/setup-deployment-guides/configurations.html#role-based-access-control))
- SSO aka Single Sign On for the Admin panel which is almost exclusively used by enterprises

In the future we are planning to open up the Admin Roles and Advanced RBAC to the community edition but we need to replace the missing enterprise level features so that we as a company can continue to pay our developers and the rest of the Strapi team (myself included).

See the response from @Aurelsicoko our Chief Product Officer on our forums regarding RBAC: https://forum.strapi.io/t/rbac-role-based-access-control-feature-discussion/433/39

---

TLDR: Open source doesn't mean free. FOSS !== OSS

FOSS is a very specific type of software that should be considered free since it stands for "Free and Open Source Software" Which generally comes from GNU based license types AKA Copyleft licenses (though even copyleft licenses don't limit your ability to sell software, it just means the source code must be available for free)

**Comment by bbarr** [comment]
This is an important discussion to be having, as I don't think a lot of folks (including myself) have an intuitive understanding of how open-source software is able to constrain free or lower-tier versions.

For example, lacking a specific feature could mean just not shipping with a particular code module. That makes sense.

However, for things like capping the number of Admin Roles - does this mean that Strapi communicates back to a centralized server? A fair amount of logic must be offloaded to this source-of-truth in order to prevent "community" users from simply updating a line of code to `const MAX_ADMIN_ROLES = 100` somewhere. 

Could someone give a quick rundown on this thread of how Strapi handles this?

Thanks!

**Comment by NickUfer** [comment]
> For example, lacking a specific feature could mean just not shipping with a particular code module. That makes sense.

This is what is happening here. There is just missing code in the webinterface in the free version to add any new admin role. That means also that you can just add admin roles manually in the database if you really need them.

**Comment by peter65374** [comment]
> Disappointing. Why do you repeat a thrillion times that strapi is free? It's not.
> 
> It's a freemium model with **limits** and a paid pro version.

agree totally.  Role permission and SSO are mandatory for almost of 90% real business. Spend a whole day and found it is not a real free open-source HCMS.

**Comment by derrickmehaffy** [comment]
FYI as I mentioned before we are making RBAC free for everyone (unlimited, except for EE users who pay per seat though community has unlimited seats) 

https://github.com/strapi/strapi/pull/16010

We hope to have this out in a release in the next few weeks.

This the paid features that replaces rbac is now:

- SSO for the admin
- Audit logs
- (soon) review workflows

We have already discussed how we can make a few more pieces open such as iterating on SSO to make the simple side (authentication) free but keep the authorization piece for EE ( things like SAML, LDAP, group syncing, eg common enterprise needs). While this isn't confirmed, it is something we are entertaining.
